qtconsole/rich_jupyter_widget.py

```diff
@@ -185,7 +183,15 @@ def _handle_display_data(self, msg):
 
     def _append_latex(self, latex, before_prompt=False, metadata=None):
         """ Append latex data to the widget."""
-        self._append_png(latex_to_png(latex, wrap=False), before_prompt, metadata)
+        supported_latex = latex.startswith('$') and latex.endswith('$')
```

Comment:
The only that `latex_to_png` can handle is Latex enclosed in dollar signs, so there's no need to check for more at the moment.